### PR TITLE
bump nginx-ingress to 1.5.1

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This updates nginx to the latest available version. This supposedly fixes a number of CVEs: https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.5.1

/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
Update nginx-ingress to 1.5.1
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: charts/nginx-ingress-controller/test/values.example.ce.yaml.out [hunk 1]
```diff
@@ -801,7 +801,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          securityContext:
+          securityContext:
```
Comment (wurbanski) [inline charts/nginx-ingress-controller/test/values.example.ce.yaml.out:804]: 😵‍💫

Comment (xrstf) [reply charts/nginx-ingress-controller/test/values.example.ce.yaml.out:804]: That would not happen if we continued to use our own manifests! :grin:

**Pull request merged**